feature/contributor/src/main/java/io/github/droidkaigi/confsched2020/contributor/ui/item/ContributorItem.kt

```diff
@@ -0,0 +1,47 @@
+package io.github.droidkaigi.confsched2020.contributor.ui.item
+
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.LiveData
+import androidx.navigation.findNavController
+import coil.api.load
+import coil.transform.CircleCropTransformation
+import com.squareup.inject.assisted.Assisted
+import com.squareup.inject.assisted.AssistedInject
+import com.xwray.groupie.databinding.BindableItem
+import io.github.droidkaigi.confsched2020.contributor.R
+import io.github.droidkaigi.confsched2020.contributor.databinding.ItemContributorBinding
+import io.github.droidkaigi.confsched2020.contributor.ui.ContributorsFragmentDirections.Companion.actionContributorsToChrome
+import io.github.droidkaigi.confsched2020.model.Contributor
+import io.github.droidkaigi.confsched2020.ui.SpeakerPlaceholderCreator
+
+class ContributorItem @AssistedInject constructor(
+    @Assisted private val contributor: Contributor,
+    private val lifecycleOwnerLiveData: LiveData<LifecycleOwner>
+) : BindableItem<ItemContributorBinding>(contributor.id.toLong()) {
+
+    override fun getLayout(): Int = R.layout.item_contributor
+
+    override fun bind(viewBinding: ItemContributorBinding, position: Int) {
+        viewBinding.contributor = contributor
+
+        val context = viewBinding.contributorImage.context
+        val placeholder = SpeakerPlaceholderCreator.create(context)
```

Comment:
You can use `lazyWithParam` method like this 👍 

```
    private val placeholder by lazyWithParam<Context, VectorDrawableCompat> { context ->
        VectorDrawableCompat.create(
            context.resources,
            R.drawable.ic_person_outline_black_32dp,
            null
        )?.apply {
            setTint(
                ContextCompat.getColor(context, R.color.speaker_icon)
            )
        }
    }
```

Reply:
Oh this is cool 😎